packages/wonder-blocks-tooltip/generated-snapshot.test.js

```diff
@@ -18,7 +18,10 @@ describe("wonder-blocks-tooltip", () => {
         const React = require("react");
 
         const example = (
-            <Tooltip content={"I'm on the right!"} placement="right">
+            <Tooltip
+                content={"This is a text tooltip on the right"}
```

Comment:
🤷‍♂️ I notice that in a few places you have extra `{...}` around strings, which can be removed!